docs/tutorial/02-draw-a-snake.md

```diff
@@ -99,47 +99,61 @@ defmodule Snake.Scene.Game do
     viewport = opts[:viewport]
 
     IO.inspect(ViewPort.info(viewport), label: "viewport info")
+
+    {:ok, :ok}
   end
 end
 ```
 
-Running the game again, we see we have some information about the size of the view port in pixels. We can use this information to compute the number of rows and columns in our grid.
+Running the game again, we see we have some information about the size of the viewport in pixels. We can use this information to compute the number of rows and columns in our grid. We'll define the default length of 32 pixels for the edge of one square and then compute how many we can fit onto the screen. As this information is necessary throughout the whole game, we store it in the `state` of the scene.
 
-When running the game you may have noticed the compiler warning us:
+```elixir
+@tile_size 32
 
-    warning: variable "arg" is unused (if the variable is not meant to be used, prefix it with an underscore)
+def init(arg, opts) do
+  viewport = opts[:viewport]
 
-In our configuration, we are passing `nil` as the first `arg` parameter to the `init/2` function of our `Game` scene, which already indicates that we're not planning to provide any information meant for further usage here. So let's follow the compiler's suggestions and prefix the argument with an underscore to indicate that we know about the parameter but don't need it:
+  {:ok, %ViewPort.Status{size: {vp_width, vp_height}}} = ViewPort.info(viewport)
 
-```elixir
-def init(_arg, opts) do
-  viewport = opts[:viewport]
+  number_of_columns = div(vp_width, @tile_size)
```

Comment:
What's the difference to using `trunc`? Just that it "includes" the computation? Makes sense I'd say 👍

Reply:
the result should be the same, but `div/2` is the native integer division function in elixir

Reply:
Perfect 💚